src/Features/Core/Navigation/ISymbolNavigationService.cs

```diff
@@ -14,7 +15,7 @@ internal interface ISymbolNavigationService : IWorkspaceService
         /// <param name="symbol">The symbol to navigate to</param>
         /// <param name="usePreviewTab">Indicates whether a preview tab should be used if the
         /// containing document is opened in a new tab. Defaults to false.</param>
-        bool TryNavigateToSymbol(ISymbol symbol, Project project, bool usePreviewTab = false);
+        bool TryNavigateToSymbol(ISymbol symbol, Project project, bool usePreviewTab = false, CancellationToken cancellationToken = default(CancellationToken));
```

Comment:
I'd prefer internal APIs not have optional cancellation tokens.  It's just an opportunity to forget them.